diff --git a/lib/StrictEventEmitter.d.ts b/lib/StrictEventEmitter.d.ts
index 12e28cfbc7012e39447ead05ecb3afe86df9da19..5acf9e2576d6e4b9cfb0ec53f403ffce608bdbff 100644
--- a/lib/StrictEventEmitter.d.ts
+++ b/lib/StrictEventEmitter.d.ts
@@ -1,4 +1,3 @@
-/// <reference types="node" />
 import { EventEmitter } from 'events';
 export declare type EventMapType = Record<string | symbol | number, any>;
 export declare class StrictEventEmitter<EventMap extends EventMapType> extends EventEmitter {