diff --git a/lib/css-tools.js b/lib/css-tools.js
index a59aae6e765ddaeee3eab6f8dda56b7c85245654..f14fba8c4d368f6c2025091bb32d9a26e0ce28ad 100644
--- a/lib/css-tools.js
+++ b/lib/css-tools.js
@@ -2,7 +2,6 @@
 
 var csstree = require('css-tree'),
   List = csstree.List,
-  stable = require('stable'),
   specificity = require('csso/lib/restructure/prepare/specificity');
 
 /**
@@ -162,7 +161,7 @@ function _bySelectorSpecificity(selectorA, selectorB) {
  * @return {Array} Stable sorted selectors
  */
 function sortSelectors(selectors) {
-  return stable(selectors, _bySelectorSpecificity);
+  return [...selectors].sort(_bySelectorSpecificity);
 }
 
 /**
diff --git a/lib/style.js b/lib/style.js
index 1873e7bd051243b97cd43462fcd50a617f282c7b..d575631beedd9cf76fb57611967c996dd3f98dbd 100644
--- a/lib/style.js
+++ b/lib/style.js
@@ -13,7 +13,6 @@
  * @typedef {import('./types').XastChild} XastChild
  */
 
-const stable = require('stable');
 const csstree = require('css-tree');
 // @ts-ignore not defined in @types/csso
 const specificity = require('csso/lib/restructure/prepare/specificity');
@@ -249,9 +248,7 @@ const collectStylesheet = (root) => {
     },
   });
   // sort by selectors specificity
-  stable.inplace(rules, (a, b) =>
-    compareSpecificity(a.specificity, b.specificity)
-  );
+  rules.sort((a, b) => compareSpecificity(a.specificity, b.specificity));
   return { rules, parents };
 };
 exports.collectStylesheet = collectStylesheet;
diff --git a/package.json b/package.json
index d827b03e25e1ad95234c83e6eec4a42485b7618f..a97f8a35d45034ceecb78372f92d46a348fece0c 100644
--- a/package.json
+++ b/package.json
@@ -105,8 +105,7 @@
     "css-select": "^4.1.3",
     "css-tree": "^1.1.3",
     "csso": "^4.2.0",
-    "picocolors": "^1.0.0",
-    "stable": "^0.1.8"
+    "picocolors": "^1.0.0"
   },
   "devDependencies": {
     "@rollup/plugin-commonjs": "^20.0.0",
diff --git a/plugins/inlineStyles.js b/plugins/inlineStyles.js
index a19f3fbd5a3ed808b9d54f6da904d2f01c3ea766..bdd5090e2adcac2f13bb27fa25bfd9015f67a483 100644
--- a/plugins/inlineStyles.js
+++ b/plugins/inlineStyles.js
@@ -9,7 +9,6 @@
 const csstree = require('css-tree');
 // @ts-ignore not defined in @types/csso
 const specificity = require('csso/lib/restructure/prepare/specificity');
-const stable = require('stable');
 const {
   visitSkip,
   querySelectorAll,
@@ -200,11 +199,13 @@ exports.fn = (root, params) => {
           return;
         }
         // stable sort selectors
-        const sortedSelectors = stable(selectors, (a, b) => {
-          const aSpecificity = specificity(a.item.data);
-          const bSpecificity = specificity(b.item.data);
-          return compareSpecificity(aSpecificity, bSpecificity);
-        }).reverse();
+        const sortedSelectors = [...selectors]
+          .sort((a, b) => {
+            const aSpecificity = specificity(a.item.data);
+            const bSpecificity = specificity(b.item.data);
+            return compareSpecificity(aSpecificity, bSpecificity);
+          })
+          .reverse();
 
         for (const selector of sortedSelectors) {
           // match selectors