diff --git a/dist/index.vue2.d.ts b/dist/index.vue2.d.ts
index 714de8e502ffd433eaeb9c088ef0f1dc1e65f303..16d9d08000c35c171d72748b4d3f9098ca271416 100644
--- a/dist/index.vue2.d.ts
+++ b/dist/index.vue2.d.ts
@@ -1,4 +1,5 @@
-import { DefineComponent, Ref } from 'vue-demi';
+import { Ref } from 'vue-demi';
+import { DefineComponent } from '@vue/runtime-dom';
 import { init, SetOptionOpts } from 'echarts/core';
 
 type InitType = typeof init;